vere/main.c

```diff
@@ -546,6 +553,66 @@ main(c3_i   argc,
     return 1;
   }
 
+  /*
+    In daemon mode, run the application as a background process.
+
+    - First setup a pipe and `fork()` the process.
+    - In the parent, read one byte from the pipe and then exit.
+    - In the child, write one byte to the pipe and then continue as per usual.
+
+    First run `fork()`,
+
+    TODO Redirect output to a file.
+    TODO Don't exit from the parent until the boot sequence finishes.
+    TODO Exit with an error code if the child process failed to boot.
+  */
+  if ( c3y == u3_Host.ops_u.dem ) {
+    int pipefd[2];
+    char buf[2] = {0,0};
+
+    if (0 != pipe(pipefd)) {
+      return 1; // TODO
+    }
+
+    fprintf(stderr, "forking\n");
+    fflush(stderr);
+    pid_t pid = fork();
+
+    if (pid) {
+      close(pipefd[1]);
+      close(0);
+      close(1);
+
+      fprintf(stderr, "parent: waiting\n");
+      fflush(stderr);
+
+      if (read(pipefd[0], buf, 1)) {
+        fprintf(stderr, "parent: urbit booted, exiting\n");
```

Comment:
This printf is pretty misleading. I'd change it to something more like `"child process started, exiting\r\n"`. Also, I think in general we want to do `\r\n` at the end of `fprintf()` calls, or else they'll show up at the end of the current line.